meshroom/nodes/aliceVision/MaskProcessing.py

```diff
@@ -0,0 +1,73 @@
+__version__ = "1.0"
+
+from meshroom.core import desc
+from meshroom.core.utils import DESCRIBER_TYPES, VERBOSE_LEVEL
+
+import os.path
+
+class MaskProcessingNodeSize(desc.DynamicNodeSize):
+    """
+    MaskProcessingNodeSize expresses a dependency to multiple input attributess to define
+    the size of a Node in terms of individual tasks for parallelization.
+    """
+    def __init__(self, param):
+        self._params = param
+
+    def computeSize(self, node):
+        
+        size = 0
+
+        for input in node.attribute(self._params).value:
+            paramName = input.getFullName()
+            param = node.attribute(paramName)
+            size = size + param.getLinkParam().node.size
```

Comment:
As we are not accumulating, we should just take the max?
size = max(size, param.getLinkParam().node.size)